modules/farm_fd2_school/src/entrypoints/FD3_Application/App.vue

```diff
@@ -105,43 +107,57 @@
       There are no {{ crop.attributes.name }} plants available for harvest.
     </div>
     <br />
-    <input
-      type="button"
-      id="harvest-submit"
-      value="Submit"
-      class="label-margin"
-      v-bind:disabled="!formValid"
-      v-on:click="submitForm"
-    />
-    <input
-      type="button"
-      id="harvest-reset"
-      value="Reset"
-      v-on:click="resetForm"
+
+    <SubmitResetButtons
+      v-bind:enableSubmit="enableSubmit"
+      v-bind:enableReset="enableReset"
+      v-on:submit="submit()"
+      v-on:reset="reset()"
+      v-on:ready="createdCount++"
     />
 
     <hr />
   </div>
 </template>
 
 <script>
+import CropSelector from '@comps/CropSelector/CropSelector.vue';
+import NumericInput from '@comps/NumericInput/NumericInput.vue';
+import CommentBox from '@comps/CommentBox/CommentBox.vue';
+import SubmitResetButtons from '@comps/SubmitResetButtons/SubmitResetButtons.vue';
 import DateSelector from '@comps/DateSelector/DateSelector.vue';
 import * as farmosUtil from '@libs/farmosUtil/farmosUtil';
 export default {
   components: {
     DateSelector,
+    CommentBox,
+    SubmitResetButtons,
+    NumericInput,
+    CropSelector,
   },
   data() {
     return {
       date: '2019-06-15',
-      crop: null,
+      crop: '',
       pickedPlant: null,
       quantity: 1,
       unit: null,
       comment: '',
       cropList: [],
       plantList: [],
       unitList: [],
+      createdCount: 0,
+      form: {
+        comment: null,
+        count: 1,
+        crop: null,
+      },
+      validity: {
```

Comment:
If you are going to use `validity` here then you should also use it's values in the `formValid` computed property below.